### ISSUE TITLE
Renamer should limit filename length

### ISSUE THREAD

**Issue by tigtigtigtig** [SickRage/sickrage-issues#594]
As much as an issue as a feature request. Still posting it here als it is causing postprocessing failures:

Most Filesystems have a maximum filename length of 255 Bytes. Sickrage doesn't check this so if a filename gets too long, postprocessing just fails and the reason "Filename too long" can be found in the logs.

OS: FreeBSD
commit: 66fa37c556784fa3b41a06bb3043bd854d165c10
branch: master

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by fernandog** [comment]
@tigtigtigtig can you give a long file name example ? maybe episode name is too long?

**Comment by OmgImAlexis** [comment]
@fernandog a quick search came up with [this](http://www.gamefaqs.com/boards/225-television-broadcast-tv/69118250), 265 chars for the episode name then you've got the show name, the space between them and anything else the renamer adds.

**Comment by tigtigtigtig** [comment]
Exactly, especially with multi-episode files the episode name can blow up the filename quite quickly.
